man/mustache.5.ron

```diff
@@ -484,6 +514,37 @@ Output, assuming the `article.mustache` from before:
     <h1>Yesterday</h1>
     <p>Nothing special happened.</p>
 
+**Dynamic Names**
+
+Some mustache implementations may allow the use of Dynamic Names, an
+**optional** part of the Mustache specification which allows to dynamically
+determine a tag's content at runtime, into parent tags.
+
+Here's an example on how Dynamic Names in parent tags work.
+
+Template:
```

Comment:
```suggestion
Templates:
```

Reply:
(I only added the final 's' to make it plural. No idea why GitHub chooses to highlight everything after the capital 'T' here...)

Reply:
> (I only added the final 's' to make it plural. No idea why GitHub chooses to highlight everything after the capital 'T' here...)

pretty weird yeah.  
I'm committing this suggestion now by the way!